### ISSUE TITLE
[Bug]: screen reader accessibility on Windows?

### ISSUE THREAD

**Issue by mehgcap** [PlummersSoftwareLLC/tmog-feedback#959]
### TMOG version and build

1.0.0

### Platform

Windows

### Operating-system version

Windows 11 25H2

### Hardware

Framework laptop 13, AMD 7840U

### What happened?

I am mostly unable to use TMOG, since I am a screen reader user. I cannot tab to anything, nor can I easily review information, such as by arrowing through an edit field's contents or having TMOG speak information automatically.

### Steps to reproduce

1. Enable a screen reader. I use NVDA, but Narrator will do just as well for this.
2. Open TMOG and try to do anything with just the keyboard and without relying on your monitor for output.

### Expected behavior

I hoped TMOG would be accessible.

### Frequency

Every time

### Before submitting

- [x] I searched for an existing report of this problem.
- [x] I removed secrets and unnecessary personal information from this public report.

**Comment by trypsynth** [comment]
For what its worth, I manage this in Taskmon, a much smaller also open source task manager replacement, namely focused specificlaly ons creen reader users. https://github.com/trypsynth/taskmon
